server/config.go

```diff
@@ -103,7 +101,7 @@ func NewConfig() *Config {
 
 const (
 	defaultLeaderLease     = int64(3)
-	defaultTsoSaveInterval = int64(2000)
+	defaultTsoSaveInterval = 2 * time.Second
```

Comment:
Because the least pd lease time is 3s, and when another PD becomes leader, the time may elapse 3s+. So I think here we can even use 3s too.